drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/commands/actualcommands/AbstractScenarioSimulationCommand.java

```diff
@@ -182,9 +182,6 @@ protected ScenarioGridColumn getScenarioGridColumnLocal(ScenarioSimulationBuilde
     protected CommandResult<ScenarioSimulationViolation> commonExecution(ScenarioSimulationContext context) {
         context.getScenarioGridPanel().onResize();
         context.getScenarioGridPanel().select();
-        if (undoable) {
```

Comment:
@jomarko 
If there is nothing preventing this, move all the "context.getScenarioGridPanel().setFocus(true);" here